README.md

```diff
@@ -12,8 +12,8 @@ Copy and paste the following snippet into your build .yml file:
 - name: Running OpenAPI Spec diff action
   uses: oasdiff/oasdiff-action/diff@main
   with:
-    base: 'specs/base.yaml'
-    revision: 'specs/revision.yaml'
+    base: 'specs/base/base.yaml'
+    revision: 'specs/revision/revision.yaml'
```

Comment:
Have you moved base and revision yamls?

Reply:
Appologies, I was not supposed to commit that, fixed.